app.json

```diff
@@ -32,6 +32,10 @@
      }
    ],
    "env": {
+     "SITE_URL": {
+       "description": "Access url to the instance, has to be https://.herokuapp.com without a trailing slash.",
```

Comment:
`"Base URL of the site. Has to be https://{app-name}.herokuapp.com."`

Reply:
done :-)